Makefile

```diff
@@ -4,6 +4,8 @@
 #
 ###############################################################################
 
+export TYK_VERSION := v5.0.1
```

Comment:
We found that (in the existing project) by relying on a shell environment variable to be set, it would be too easy to forget to set it in a new shell window and would revert to defaults.  So for our builds we just always define the TYK_VERSION in the Makefile - it's easy enough to change as needed.